docs/user-guide/references.rst

```diff
@@ -17,43 +17,78 @@ Glossary
     1D Analysis
       1D analysis or spectral analysis where data are reduced to a simple 1D
       geometry along the reconstructed energy axis. In Cherenkov astronomy,
-      this is classically performed with a OFF background measurement.
+      this is classically performed with a OFF background measurement [Piron2001]_.
 
     3D Analysis
       3D analysis or cube analysis, where data are reduced to a 3D cube with
-      spatial coordinates and energy axes. In gammapy, these cube are represented
-      by `Map` objects (see :ref:`maps`) and contained in a `MapDataset` object.
+      spatial coordinates and energy axes [Stewart2009]_. In Gammapy, these cube are
+      represented by `Map` objects (see :ref:`maps`) and contained in a `MapDataset` object.
 
     Aeff
-      Short for effective area: it is the IRF representing the detector collection
-      area. See :ref:`irf-aeff`.
+      Short for "effective area": it is the IRF representing the detector collection
+      area. See :ref:`irf-theory` and :ref:`irf-aeff`.
+
+    Bkg
+      Short for "background": it is the IRF representing the residual background rate
+      per solid angle. See :ref:`irf-theory` and :ref:`irf-bkg`.
 
     Cash
       The cash statistic is a Poisson fit statistic usually used when signal and
-      background can be modeled. It is defined as :math:`2 \times log(L)` See
+      background can be modeled [Cash1979]_. It is defined as :math:`2 \times log(L)`. See
       :ref:`cash` in :ref:`fit statistics <fit-statistics>`.
 
     Dataset
       In Gammapy a dataset bundles the data, IRFs, model and a likelihood function.
       Based on the model and IRFs the predicted number of counts are computed and
-      compared to the measured counts using the likelihood.
+      compared to the measured counts using the likelihood. See :ref:`datasets`
+
+    DL3
+      Short for "data level 3": it is used to mention or describe science-ready data, ie
+      the event list (with basically a Time, an arrival direction and an energy) and the
+      four IRFs (Aeff, EDisp, PSF, Bkg). See :ref:`data flow <data_flow>`.
```

Comment:
Maybe its better to not explicitly mention the 4 IRFs (eg: PSF will be missing for ones with theta sq cut).
Rather, "the event list containing reconstructed parameters (arrival time, direction energy, etc) and the associated IRFs"

Reply:
Also, link to the GADF ?